packages/typespec-client-generator-core/src/decorators.ts

```diff
@@ -77,8 +78,9 @@ function setScopedDecoratorData(
   const targetEntry = context.program.stateMap(key).get(target);
   // If target doesn't exist in decorator map, create a new entry
```

Comment:
could you check the logic of target exists? i think that branch does not handle the scope with `,`